### PR TITLE
Updates the app with new branding and colors

### PR DESCRIPTION
**Pull request by designmatty**
## Overview

Initial rebranding effort for the application. I also noticed issues with scrolling in the sidebar and I fixed those as well.

Connects #238
- [x] Updated logo
- [x] Updated colors
- [x] Updated font. Waiting on required code, before we can implement.

## Demo
Screen Shot 2019-03-13 at 11 48 46 AM

### CODE REVIEW

File: src/app/src/util/constants.oarmap.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import env from './env';
 
 export const MAPBOX_TOKEN = env('REACT_APP_MAPBOX_TOKEN');
 
-export const CIRCLE_COLOR = '#3d2f8c';
+export const CIRCLE_COLOR = '#0427a4';
```
Comment (jwalgran) [inline src/app/src/util/constants.oarmap.js]: This file has been deleted in the switch to Google Maps PR. https://github.com/open-apparel-registry/open-apparel-registry/pull/284 If that is merged first you may run into a conflict. 

Once the Google Maps PR is merged we will need some follow up design work to make markers and clusters that match the branding.

Comment (kellyi) [reply src/app/src/util/constants.oarmap.js]: We should move these values into the `COLOURS.js` file and get them from there.

**Comment by jwalgran** [comment]
This isn't a regression, but I just noticed that the alignment of the menu bar items is off

Screen Shot 2019-03-13 at 10 33 33 AM

**Comment by designmatty** [comment]
I think the navbar alignment is due to using different components for in app links and external links. Is there a way of utilizing the same component?

**Comment by jwalgran** [comment]
There are a few different components in that navbar. Plan anchors, react-router Links, and dropdown menus.

Wrapping some items with span tags works around the issue. Is this too gross?

```diff
diff --git a/src/app/src/components/Navbar.jsx b/src/app/src/components/Navbar.jsx
index 4bc8d16..d07bd3c 100644
--- a/src/app/src/components/Navbar.jsx
+++ b/src/app/src/components/Navbar.jsx
@@ -42,30 +42,36 @@ export default function Navbar() {
                         marginLeft: 'auto',
                     }}
                 >
-                    <Link
-                        to="/"
-                        href="/"
-                        className="navButton"
-                    >
-                        HOME
-                    </Link>
+                    <span>
+                        <Link
+                            to="/"
+                            href="/"
+                            className="navButton"
+                        >
+                            HOME
+                        </Link>
+                    </span>
                     <NavbarDropdown title="ABOUT" links={aboutLinks} />
-                    <a
-                        target="_blank"
-                        className="navButton"
-                        rel="noopener noreferrer"
-                        href="https://info.openapparel.org/faq/"
-                    >
-                        FAQs
-                    </a>
-                    <a
-                        target="_blank"
-                        className="navButton"
-                        rel="noopener noreferrer"
-                        href="https://info.openapparel.org/apiinstructions"
-                    >
-                        API
-                    </a>
+                    <span>
+                        <a
+                            target="_blank"
+                            className="navButton"
+                            rel="noopener noreferrer"
+                            href="https://info.openapparel.org/faq/"
+                        >
+                            FAQs
+                        </a>
+                    </span>
+                    <span>
+                        <a
+                            target="_blank"
+                            className="navButton"
+                            rel="noopener noreferrer"
+                            href="https://info.openapparel.org/apiinstructions"
+                        >
+                            API
+                        </a>
+                    </span>
                     <span>
                         <Link
                             className="navButton"

```

**Comment by kellyi** [comment]
> Is there a way of utilizing the same component?

Interesting! I had thought that the `Link` elements were just `a` elements in the end but they may not be. Is there some global CSS rule applied to regular `a` elements that the `Link`s are either ignoring or overriding?

**Comment by kellyi** [comment]
The `Link` elements may have their own implicit span or div already also.

File: src/app/src/util/constants.oarmap.js (hunk 1 above)
Comment (kellyi) [reply src/app/src/util/constants.oarmap.js]: I moved these during a rebase.

**Comment by kellyi** [comment]
Looking at this now and the elements that are offcenter are `div`s that open menus rather than `Link` or `a`:

Screen Shot 2019-03-13 at 2 54 28 PM

I think we can just give those divs the same styling as the links. Right now they don't seem to have any intrinsic style.

**Comment by designmatty** [comment]
**Added a favicon**
connects #157 
Screen Shot 2019-03-14 at 12 34 28 PM

**Fixes navigation design and alignment**
- uniform line heights
- uniform hover states
Screen Shot 2019-03-14 at 12 34 36 PM

Note: I wasn't able to completely get the log out button to look identical due to the nature of how `<button>` is rendered vs `<a>`
Screen Shot 2019-03-14 at 12 34 41 PM

**Pull request merged**